src/core/qfieldcloudconnection.cpp

```diff
@@ -513,3 +518,62 @@ QFieldCloudConnection::CloudError::CloudError( QNetworkReply *reply )
 
   mMessage = errorMessage;
 }
+
+void QFieldCloudConnection::uploadPendingAttachments()
+{
+  if ( mUploadingAttachments )
+    return;
+
+  QMultiMap<QString, QString> attachments = QFieldCloudUtils::getPendingAttachments();
+  if ( attachments.isEmpty() )
+  {
+    return;
+  }
+
+  mUploadCount = attachments.size();
+  QMultiMap<QString, QString>::const_iterator it = attachments.constBegin();
+  while ( it != attachments.constEnd() )
+  {
+    if ( !QFileInfo( it.value() ).exists() )
+    {
+      QFieldCloudUtils::removePendingAttachment( it.key(), it.value() );
```

Comment:
Is this situation expected? If not, log a warning?

Reply:
+1 for logging and a comment that this is normal when the user deletes the QFieldCloud project from their device.